.github/workflows/publish-cli.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: set version env variable
-        run: echo "CRATE_VERSION=$(cat Cargo.toml | sed -n 's/.*version = "\([^"]*\)".*/\1/p' | head -1)" >> $GITHUB_ENV
+        run: echo "CRATE_VERSION=$(cat cli/Cargo.toml | sed -n 's/.*version = "\([^"]*\)".*/\1/p' | head -1)" >> $GITHUB_ENV
       - name: create release
         id: create_release
         uses: actions/create-release@v1.1.0
```

Comment:
I can't leave a comment on line 21, but shouldn't the tag name be `cli-wallet-v${{ env.CRATE_VERSION }}` too?

Reply:
Yes makes sense

Reply:
Ok, I can add that in this PR later today